packages/projection/package.json

```diff
@@ -9,6 +9,7 @@
   "module": "dist/esm/index.js",
   "exports": {
     ".": {
+      "development": "./src/index.ts",
```

Comment:
If not too difficult, would be great to extract hot reload changes into a separate commit

Reply:
Maybe I misunderstand the purpose of hot reload: isn't it to restart the provider server when ts files change? If that's correct, then it could be used for non-websocket provider server development too. :thinking:

Reply:
Yes, it is my intention, but before applying to all other container I preferred to have the team approval (in this PR), next propagating to other containers with a distinct ticket and PR.

Reply:
Ah I see, [this](https://github.com/input-output-hk/cardano-js-sdk/pull/1352/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R126) would belong in feature PR as it's adding a new type of server.

If we extract it, then maybe we can just add an npm script to run tsx for the existing provider server. Then team can review it in isolation.

Reply:
Decoupled #1354 to handle this